### ISSUE TITLE
NFT detail page: Loading state

### ISSUE THREAD

**Issue by piersss** [airswap/airswap-marketplace#51]

**Comment by 2xAST** [comment]
+1 for skeletons. Do you need them as a design for reference?
I think we could reuse the style and animation we have on the web-app

**Comment by Goddak** [comment]
@2xAST - Do you have a design from web showing the style?

**Comment by piersss** [comment]
@Goddak In the swap feature when it's searching for quotes.

**Comment by piersss** [comment]
> +1 for skeletons. Do you need them as a design for reference? I think we could reuse the style and animation we have on the web-app

Yes please make a design for reference. We can probably reuse it for other pages.
